_data/testimonials.yml

```diff
@@ -9,3 +9,7 @@
 - name: Carlos Pedroza
   username: carlospedroza27
   text: Makeroid is merging the ability to create awesome android applications without writing a single line of code and the ability to make a professional app in a stunning way.
+
```

Comment:
I think that due to testimonials design there's a limit of 3
But I'll see if we can display randomly

Reply:
@pavi2410 made it a carousel, have you seen it?

https://pavi2410.tk/makeroid.github.io/#testimonials